threadit-web/src/pages/PostSpool/PostSpool.tsx

```diff
@@ -40,8 +44,13 @@ export default function PostSpool() {
                                     <Input disabled={lockInputs} size='md' value={title} onChange={(e) => setTitle(e.target.value)} />
```

Comment:
I think the input should remain the width of the container and the interests should populate above it. Is that possible?

Reply:
I can take a look. since its a custom template, I may rely on noah to try and give me a hand with that one in the future though